### PR TITLE
[MM-29493,  MM-29494] Migrate changeCSS() to CSS variable in utils/ut…

### PR DESCRIPTION
**Pull request by deanwhillier**
#### Summary
ApplyTheme Campaign: This PR completes the merge of the theme.onlineIndicator commits from the campaign branch.

Dev approval for the changes was completed on the original PR(s), so we are just looking for rubber stamp approvals by dev's to satisfy merge requirements.

As QA was skipped on PR merges into campaign branches, this PR will need proper QA before merging. See below for details on what to QA for this PR.

#### Ticket Link

* https://github.com/mattermost/mattermost-server/issues/15881
* https://github.com/mattermost/mattermost-server/issues/15882

#### Related Pull Requests
See tickets for related pull requests

### CODE REVIEW

**Comment by deanwhillier** [comment]
@jgilliam17, this is is like the away indicator but for online indicator 🙂

File: sass/components/_status-icon.scss [hunk 1]
```diff
@@ -208,3 +208,14 @@
...
+.app__body{
+    .status{
+        &.status--online{
+            color: var(--online-indicator);
+        }
+        .online--icon{
+            fill: var(--online-indicator);
+        }
+    }
+}
```
Comment (nevyangelova) [inline sass/components/_status-icon.scss:221]: nit: new line

**Review by nevyangelova — APPROVED** [review]
LGTM -one nit pick comment

File: sass/components/_status-icon.scss (hunk 1 above)
Comment (deanwhillier) [inline sass/components/_status-icon.scss:221]:
```suggestion
}

```

**Review by hmhealey — APPROVED** [review]

**Comment by hmhealey** [comment]
This is going to need a merge since the away indicator one touches the same files

**Review by jgilliam17 — APPROVED** [review]
Thanks @deanwhillier 
Tested, looks good to merge.
- Verified `online` status indicator color in 4 main themes.

**Pull request merged**